report_xlsx/models/ir_report.py

```diff
@@ -1,7 +1,12 @@
 # Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, fields, models
+import logging
```

Comment:
Please don't hammer log with a lot of extra lines. Is the standard doing the same for the attachment?

Reply:
Hi yes this code is similar to standard in [`base/models/ir_actions_report.py > _render_qweb_pdf`](https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/models/ir_actions_report.py#L808) where they do add warning / info log lines

Reply:
OK, I'm not very fan of excessive logging, but let's mimic the standard.